libraries/core/src/main/kotlin/com/bumble/appyx/core/navigation/transition/CombinedHandler.kt

```diff
@@ -24,5 +25,7 @@ class CombinedHandler<T, S>(
 }
 
 @Composable
-fun <T, S> rememberCombinedHandler(handlers: List<ModifierTransitionHandler<T, S>>): ModifierTransitionHandler<T, S> =
+fun <T, S> rememberCombinedHandler(
+    handlers: ImmutableList<ModifierTransitionHandler<T, S>>
+): ModifierTransitionHandler<T, S> =
     remember { CombinedHandler(handlers = handlers) }
```

Comment:
```suggestion
    remember(handlers) { CombinedHandler(handlers = handlers) }
```

Reply:
That's scope creep, though I guess this is already a slight API change =P

Reply:
done